global_helpers/panther_base_helpers.py

```diff
@@ -153,3 +155,7 @@ def deep_get(dictionary, *keys, default=None):
     return reduce(
         lambda d, key: d.get(key, default)
         if isinstance(d, Mapping) else default, keys, dictionary)
+
+
+FIPS_ENABLED = os.getenv('ENABLE_FIPS', '').lower() == 'true'
+FIPS_SUFFIX = '-fips.' + os.getenv('AWS_REGION', '') + '.amazonaws.com'
```

Comment:
Since we're not using these in this file and not importing them we can drop them I think.